tutorials/install.md

```diff
@@ -85,6 +95,45 @@ Build and install as follows:
     make -j4
     sudo make install
 
+#### Windows
+
+Open a Visual Studio Command Prompt (search for "x64 Native Tools Command Prompt for VS 2019" in the Windows search near the Start button).
```

Comment:
FYI PR https://github.com/conda-forge/vc-feedstock/pull/19 added the package `vs2019_win-64`, that should source automatically the necessary variables of VS2019, but to be honest I tried it now and it is not working correctly for me, so probably sticking to this well known method is a good idea for now.

Reply:
"feedstock" is a new word to me. Is the idea that with that package, Visual Studio can be installed via something like `conda install vc vs2019_win-64`, instead of from the Microsoft downloads? And then instead of using the VS Command Prompt, people can use the Anaconda prompt?

Reply:
Sorry for the jargon. In conda, the "feedstock" is the repository that contains the metadata and the scripts to generate a conda package for a given library (see https://conda-forge.org/#about for more info).

The `vs2019_win-64` conda package does not install Visual Studio 2019, but it just automatically invokes the script to set the environment variables set in the `VS Command Prompt` when a conda environment containg it is activated. However, as I was writing in the previous comment is not working correctly (at least for me). In any case if you use the `Visual Studio 2019 16` Generator (as you are doing implicitly now in the docs), instead of the Ninja or NMake generator, it is not necessary to use it as CMake will handle all that configuration automatically.

Reply:
Thanks for the explanation! I understood half of that, but sounds good.

Reply:
Feel free to ask for the point in which I was not clear. Re-reading it, I think it would be helpful to point out that the default CMake generator in Windows is the latest Visual Studio one if found, so following the instructions the build will default to use https://cmake.org/cmake/help/v3.19/generator/Visual%20Studio%2016%202019.html .